src/exodus/test/sigmadb_tests.cpp

```diff
@@ -348,4 +375,64 @@ BOOST_AUTO_TEST_CASE(get_anonimity_group_by_iterator)
     BOOST_CHECK(pubs == std::vector<exodus::SigmaPublicKey>(anonimityGroupList.begin(), anonimityGroupList.end()));
 }
 
-BOOST_AUTO_TEST_SUITE_END()
\ No newline at end of file
+BOOST_AUTO_TEST_CASE(group_size_default)
+{
+    std::string testingDefaultFile = "MP_txlist_groupsize_test";
+
+    auto CMPMintListProxy = GetCMPMintList(testingDefaultFile, 0);
```

Comment:
No deletion for `CMPMintListProxy`, thus memory leak.

Reply:
use `std::unique_ptr` instead then this one should be solved.